### ISSUE TITLE
`beta` parameters in `skimage.filters.{frangi/hessian}` are poorly documented

### ISSUE THREAD

**Issue by soupault** [scikit-image/scikit-image#2166]
Parameters `beta1, beta2` docstrings (see https://github.com/scikit-image/scikit-image/blob/master/skimage/filters/_frangi.py#L20-L23, https://github.com/scikit-image/scikit-image/blob/master/skimage/filters/_frangi.py#L91-L94, https://github.com/scikit-image/scikit-image/blob/master/skimage/filters/_frangi.py#L148-L151) of the corresponding filters require some enhacement.
Without a detailed description and recommendations on values' choice those former are of zero use.

**Comment by soupault** [comment]
Should also adjust figsize in
http://scikit-image.org/docs/dev/auto_examples/filters/plot_frangi.html#sphx-glr-auto-examples-filters-plot-frangi-py.

**Comment by jni** [comment]
@Hiyorimi could you tackle this? Or just write a description here if you can't be bothered with a PR...? =)

**Comment by Hiyorimi** [comment]
@jni

Beta1 controls edge thickness (better to scale as e at first, i.e. 0.1, 1, 10)
Beta2 controls noise tolerance 

Should I submit a PR ?

**Comment by jni** [comment]
@Hiyorimi 

> Should I submit a PR

always appreciated. =)

**Comment by jni** [comment]
If possible, a slightly more verbose description is better, eg.:

```
beta1 : float, optional
    Controls edge thickness. Higher beta looks for thicker edges. Best to vary on a log-scale, e.g. 0.1, 1, 10...
beta2 : float, optional
    Controls noise tolerance. Higher beta is more tolerant to noise but ... (there must be some caveat?)
```

**Comment by stefanv** [comment]
Notes form the paper:

- beta1 adjusts the filter's sensitivity to deviation from a blob-like structure.
- beta2 adjusts the filter's sensitivity to areas of high variance/texture/structure.

**Comment by stefanv** [comment]
@jni w.r.t. beta2, more tolerant to noise, but less sensitive to pick up actual structure

**Comment by soupault** [comment]
Closed via #2382.
